data/json/items/armor/power_armor.json

```
@@ -1,695 +1,447 @@
 [
   {
-    "id": "depowered_armor",
-    "type": "ARMOR",
+    "abstract": "exoskeleton",
+    "type": "TOOL_ARMOR",
     "category": "armor",
-    "name": { "str": "salvaged power armor" },
-    "//": "These are much cheaper than powered suits, because they're extremely heavy and impractical.  The good stuff is gone!",
-    "description": "This used to be a powered exoskeleton with a set of heavy armor plates, made wearable by the assistance of servos.  Now it's just the armor plating and a stripped-down chassis: as one might expect, it's a lot harder to wear.",
-    "weight": "12214 g",
-    "volume": "25 L",
-    "price": 70000,
-    "price_postapoc": 15000,
-    "to_hit": 1,
-    "material": [ "steel" ],
+    "name": { "str": "abstract exoskeleton" },
     "symbol": "[",
-    "looks_like": "power_armor_basic",
-    "color": "light_gray",
+    "price": 10000000,
+    "price_postapoc": 15000,
+    "flags": [ "STURDY", "WATERPROOF", "RAINPROOF", "SUN_GLASSES", "USE_UPS", "NO_UNLOAD", "COMBAT_TOGGLEABLE" ],
+    "relic_data": { "passive_effects": [ { "id": "ench_climate_control_warm" } ] },
+    "warmth": 40,
+    "environmental_protection": 10,
+    "ammo": "battery",
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
         "rigid": true,
-        "max_contains_volume": "4 L",
-        "max_contains_weight": "30 kg",
-        "description": "Compartment for an UPS.",
+        "description": "Compartment for a UPS.",
+        "flag_restriction": [ "IS_UPS" ],
+        "max_contains_volume": "2.51 L",
```

```suggestion
        "max_contains_volume": "2510 ml",
```

If you need more precision than whole liter, use milliliters.